tests/lax_numpy_test.py

```diff
@@ -6090,7 +6090,7 @@ def test_lax_numpy_docstrings(self):
     # Test that docstring wrapping & transformation didn't fail.
 
     unimplemented = ['fromfile', 'fromiter']
-    aliases = ['abs']
+    aliases = ['abs', 'acos', 'acosh', 'asin', 'asinh', 'atan', 'atanh', 'atan2', 'bitwise_right_shift']
```

Comment:
It looks like the base for this PR is outdated. Can you rebase on an updated main branch?